### PR TITLE
Updates to HyPhy wrappers

### PR DESCRIPTION
**Pull request by davebx**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR updates an existing tool or tool collection
- Update tool version to 2.5.31
- Support fasta.gz and nexus inputs where relevant
- Macro-ify input and output files

### CODE REVIEW

File: tools/hyphy/hyphy_fubar.xml [hunk 1]
```diff
@@ -3,20 +3,24 @@
     <description>Fast Unconstrained Bayesian AppRoximation</description>
     <macros>
         <import>macros.xml</import>
+        <token name="@OPERATION@">FUBAR</token>
```
Comment (bernt-matthias) [inline tools/hyphy/hyphy_fubar.xml:6]: Where are those tokens used?

Comment (davebx) [reply tools/hyphy/hyphy_fubar.xml:6]: https://github.com/galaxyproject/tools-iuc/pull/3598/files#diff-90bfa2f2d9fe3174844eaf5edc72bdd6418e85a2cf68ee69846b80244b8cd4b9R138

Comment (bgruening) [reply tools/hyphy/hyphy_fubar.xml:6]: I'm nervous about the case-sensitive operations and the different meaning :)
But I guess this is ok.

File: tools/hyphy/hyphy_absrel.xml [hunk 2]
```diff
@@ -3,14 +3,17 @@
     <description>adaptive Branch Site Random Effects Likelihood</description>
     <macros>
         <import>macros.xml</import>
+        <token name="@OPERATION@">aBSREL</token>
```
Comment (bgruening) [inline tools/hyphy/hyphy_absrel.xml:6]: should the a also be capital?

Comment (davebx) [reply tools/hyphy/hyphy_absrel.xml:6]: No, that's how it's referenced in the underlying code

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Will merge, but stop to have it easier to review the other PR :)

**Pull request merged**